app/views/categories/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="container mt-2">
+<% if notice.present? %>
+  <div class="alert alert-success text-center" role="alert">
+    <%= notice %>
+  </div>
+<% end %>
+<header class="items-header">
+  <h1 class="categories-title">Categories</h1>
+</header>
+
+<% @categories.each do |category| %>
+<div class="category-card">
+    <%= image_tag category.icon, height: 50 , alt: 'icon', class: "category-icon"%>
+<div class="card-body">
+<%= link_to category.name, category_payments_path(category), class: "card-title h5 text-decoration-none" %>
+</div>
+<p class="card-text"><span class="font-weight-bold">Total:</span> <%= number_to_currency(@total_amounts[category.id]) %></p>
+</div>
+<% end %>
+
+<%= link_to 'Add a new category', new_category_path, class: "btn add-new mb-4 d-block " %>
```

Comment:
- [ ] [OPTIONAL] It would be nice to stick to design guide, your add category button should be at the bottom of d page as it is in the design and also font sizes do not match
your design :point_down: 
![Screenshot from 2023-05-06 14-27-25](https://user-images.githubusercontent.com/66097371/236627320-ef821e2a-13e2-4011-944b-adfeb9169dd8.png)
design guide :point_down: 
![Screenshot from 2023-05-06 13-54-49](https://user-images.githubusercontent.com/66097371/236627333-5fcb6b56-4c95-4285-86f1-142f6cd952f4.png)

Reply:
Thanks @VICTOR-CHUKWU, for the review!! I, let me do it now!!